### PR TITLE
fix: Renamed etxn, tx, txn terminology to expense in view expense, mileage and per diem pages

### PR DESCRIPTION
**Pull request by OmkarJ13**
### <samp>🤖 Generated by Copilot at 48bb322</samp>

Renamed variables and observables related to expenses in the view-expense, view-mileage, and view-per-diem pages and their spec files, to use the term `expense` instead of `etxn` or `txn`. This is to improve the consistency and clarity of the code and the terminology. Added TODO comments to mark pending changes in the navigation footer component.

### <samp>🤖 Generated by Copilot at 48bb322</samp>

> _`etxn` to `expense`_
> _Renaming for clarity_
> _Spring cleaning the code_

### <samp>🤖 Generated by Copilot at 48bb322</samp>

*  Rename `etxn` and `etxnCurrencySymbol` to `expense` and `expenseCurrencySymbol` in view-expense, view-mileage, and view-per-diem pages and their spec files ([link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL11-R14), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL81-R81), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL628-R632), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL642-R646), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL477-R477), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL498-R498), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL513-R513), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL602-R602), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL614-R614), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL866-R867), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L90-R96), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L124-R124), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L289-R299), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L306-R312), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L334-R337), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L429-R431), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL11-R15), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL74-R74), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL416-R420), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL430-R434), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L478-R478), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L501-R501), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L511-R511), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L651-R651), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L665-R665), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L697-R697), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L717-R717), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L812-R812), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L833-R833), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L854-R854), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L1004-R1005), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL79-R81), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL87-R87), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL99-R99), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL308-R318), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL325-R331), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL356-R359), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1f66dc8a678505b425a50f4d2471159dc2b8c932f372ab63451b24d6db9a19daL435-R439), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L11-R15), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L72-R72), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L309-R313), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L323-R327), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-aa7434f7715887f5188bf634482b759f816a20bd7f5d51ef219f9754ec71a057L315-R315), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-aa7434f7715887f5188bf634482b759f816a20bd7f5d51ef219f9754ec71a057L660-R665), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L77-R79), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L85-R85), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L95-R95), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L194-R206), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L211-R221), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L237-R244), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L329-R337))
* Rename `txnFields$` to `expenseFields$` in view-expense, view-mileage, and view-per-diem pages and their spec files ([link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL628-R632), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL642-R646), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL477-R477), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL513-R513), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-3193ca2e160d90be32f2ebdc92d004542d0ad5042080accb5d69c8835a83f1feL614-R614), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L124-R124), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L289-R299), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L306-R312), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-717fc694a7c204cc8c4c0a0534c256936252ffa212d988754376936a1fe543a1L334-R337), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL416-R420), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL430-R434), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L478-R478), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L501-R501), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L511-R511), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L665-R665), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L697-R697), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L717-R717), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L812-R812), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L833-R833), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-fcebf8e22c0eff0e4b6512cabb52653cc8eb69475309a8c02c66ab8840563f55L854-R854), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L309-R313), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L323-R327), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-aa7434f7715887f5188bf634482b759f816a20bd7f5d51ef219f9754ec71a057L315-R315), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L95-R95), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L194-R206), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L211-R221), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-b66c53e7e9d2eaf1447540624a0e92ad7ea25c5299edf4e5720110afb75cec90L237-R244))
* Add TODO comments to indicate that the etxn terminology should be renamed to expense in the `navigation-footer` component, which is used in view-expense, view-mileage, and view-per-diem pages ([link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-38bc8efb9ce49f72845a76cfe9045d647245f157542e5281df732a05304ba5dcL680-R684), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-52c28d7749dea1c87c22dc8b0949ba9eae9a53488b0861818624029d1c09032bL464-R468), [link](https://github.com/fylein/fyle-mobile-app/pull/2553/files?diff=unified&w=0#diff-1832dda9cae0b28d660ccdbc9c69cb37d3bb461cc699eea8d9561beee4f29ca0L357-R361))

## Clickup
https://app.clickup.com

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/fyle/view-expense/view-expense.page.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
     <ion-title
       mode="md"
       class="page-title"
-      [ngClass]="{'view-expense--title': numEtxnsInReport > 1, 'pl-0': isDeviceWidthSmall}"
+      [ngClass]="{'view-expense--title': numExpensesInReport > 1, 'pl-0': isDeviceWidthSmall}"
       >View Expense
```
Comment (SowmyaFyle) [inline src/app/fyle/view-expense/view-expense.page.html:12]: could we rename to something like `expenseCountInReport` or `reportExpenseCount`

File: src/app/fyle/view-expense/view-expense.page.html [hunk 2]
```diff
@@ -8,10 +8,10 @@
...
-      <p class="view-expense--subtitle" *ngIf="numEtxnsInReport > 1">
-        Showing {{ activeEtxnIndex + 1 }} of {{ numEtxnsInReport }} expenses
+      <p class="view-expense--subtitle" *ngIf="numExpensesInReport > 1">
+        Showing {{ activeExpenseIndex + 1 }} of {{ numExpensesInReport }} expenses
       </p>
```
Comment (SowmyaFyle) [inline src/app/fyle/view-expense/view-expense.page.html:15]: same as above for `numExpensesInReport`

File: src/app/fyle/view-expense/view-expense.page.html [hunk 3]
```diff
@@ -677,8 +677,9 @@
   </ng-template>
 </ion-content>
 
+<!-- TODO: Rename etxn terminology to expense in navigation footer component -->
 <app-navigation-footer
-  *ngIf="numEtxnsInReport > 1"
-  [activeEtxnIndex]="activeEtxnIndex"
-  [numEtxnsInReport]="numEtxnsInReport"
+  *ngIf="numExpensesInReport > 1"
+  [activeEtxnIndex]="activeExpenseIndex"
```
Comment (SowmyaFyle) [inline src/app/fyle/view-expense/view-expense.page.html:683]: rename

File: src/app/fyle/view-mileage/view-mileage.page.html [hunk 4]
```diff
@@ -461,8 +461,9 @@
...
 
+<!-- TODO: Rename etxn terminology to expense in navigation footer component -->
 <app-navigation-footer
-  *ngIf="numEtxnsInReport > 1"
-  [activeEtxnIndex]="activeEtxnIndex"
-  [numEtxnsInReport]="numEtxnsInReport"
+  *ngIf="numExpensesInReport > 1"
+  [activeEtxnIndex]="activeExpenseIndex"
+  [numEtxnsInReport]="numExpensesInReport"
 ></app-navigation-footer>
```
Comment (SowmyaFyle) [inline src/app/fyle/view-mileage/view-mileage.page.html:469]: rename

File: src/app/fyle/view-per-diem/view-per-diem.page.html [hunk 5]
```diff
@@ -306,11 +306,11 @@
...
     <ng-container *ngIf="projectDependentCustomProperties$ | async as projectDependentCustomProperties">
-      <ng-container *ngIf="txnFields$ | async as txnFields">
-        <ng-container *ngIf="txnFields?.project_id?.length">
+      <ng-container *ngIf="expenseFields$ | async as expenseFields">
+        <ng-container *ngIf="expenseFields?.project_id?.length">
           <app-view-dependent-fields
             parentFieldType="PROJECT"
-            [parentFieldName]="txnFields.project_id[0]?.field_name"
+            [parentFieldName]="expenseFields.project_id[0]?.field_name"
             [parentFieldValue]="perDiemExpense.project?.name"
```
Comment (SowmyaFyle) [inline src/app/fyle/view-per-diem/view-per-diem.page.html:314]: ` project_id?.[0].id?` ignore if `expenseFields.project_id[0]?.field_name` is apt for this case

File: src/app/fyle/view-per-diem/view-per-diem.page.ts [hunk 6]
```diff
@@ -326,11 +330,11 @@ export class ViewPerDiemPage {
...
     this.updateFlag$.next(null);
 
-    const etxnIds =
+    const expenseIds =
       this.activatedRoute.snapshot.params.txnIds &&
       (JSON.parse(this.activatedRoute.snapshot.params.txnIds as string) as string[]);
-    this.numEtxnsInReport = etxnIds.length;
-    this.activeEtxnIndex = parseInt(this.activatedRoute.snapshot.params.activeIndex as string, 10);
+    this.numExpensesInReport = expenseIds.length;
+    this.activeExpenseIndex = parseInt(this.activatedRoute.snapshot.params.activeIndex as string, 10);
```
Comment (SowmyaFyle) [inline src/app/fyle/view-per-diem/view-per-diem.page.ts:337]: rename

**Review by SowmyaFyle — APPROVED** [review]
minor comments

File: src/app/fyle/view-per-diem/view-per-diem.page.html (hunk 5 above)
Comment (OmkarJ13) [reply src/app/fyle/view-per-diem/view-per-diem.page.html:314]: expenseFields.project_id[0]?.field_name This is apt 👍

File: src/app/fyle/view-expense/view-expense.page.html (hunk 3 above)
Comment (OmkarJ13) [reply src/app/fyle/view-expense/view-expense.page.html:683]: The navigation-footer component inputs are renamed in a separate PR, hence added a comment above

File: src/app/fyle/view-expense/view-expense.page.ts [hunk 7]
```diff
@@ -426,9 +426,9 @@ export class ViewExpensePage {
     });
 
     if (this.activatedRoute.snapshot.params.txnIds) {
-      const etxnIds = JSON.parse(this.activatedRoute.snapshot.params.txnIds as string) as string[];
-      this.numEtxnsInReport = etxnIds.length;
-      this.activeEtxnIndex = parseInt(this.activatedRoute.snapshot.params.activeIndex as string, 10);
+      const expenseIds = JSON.parse(this.activatedRoute.snapshot.params.txnIds as string) as string[];
```
Comment (OmkarJ13) [inline src/app/fyle/view-expense/view-expense.page.ts:429]: This `txnIds` query param will clean up separately once everything is, as it needs cleanup from many other pages too.

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**